src/agda.sty

```diff
@@ -594,7 +594,7 @@
 %
 % The implementation is based on plainhscode in lhs2TeX's
 % polycode.fmt, written by Andres Löh.
-\NewEnviron{code}[1][]{%
+\NewEnviron{Code}[1][]{%
```

Comment:
I changed this because `code` is already defined in the ctfp setup/config.  Actually, I'm not sure we even need the `agda.sty` file (unless we want to figure out how to incorporate the latex output from the command `agda --latex filename.agda` into the document, in place of the Agda code snippets.